src/seedu/addressbook/commands/DeleteCommand.java

```diff
@@ -30,8 +33,8 @@ public CommandResult execute() {
         try {
             final ReadOnlyPerson target = getTargetPerson();
             addressBook.removePerson(target);
-            return new CommandResult(String.format(MESSAGE_DELETE_PERSON_SUCCESS, target));
-
+            List<ReadOnlyPerson> allPersons = addressBook.getAllPersons().immutableListView();
+            return new CommandResult(String.format(MESSAGE_DELETE_PERSON_SUCCESS, target), allPersons);
```

Comment:
Another way of doing this is by "calling" list command in the Main.java. Refer to https://github.com/nus-cs2103-AY1718S2/addressbook-level2/pull/606/files

Reply:
Thanks for the suggestion!